lib/android-tools-info.ts

```diff
@@ -159,6 +159,13 @@ export class AndroidToolsInfo implements IAndroidToolsInfo {
 
 	public validateJavacVersion(installedJavaVersion: string, options?: { showWarningsAsErrors: boolean }): IFuture<boolean> {
 		return ((): boolean => {
+			// Check for min supported Java version
+			let minJavaVersion = "1.8";
+			if (parseInt(installedJavaVersion.substr(0, 3).replace(".", "")) < parseInt(minJavaVersion.replace(".", ""))) {
```

Comment:
This could fail on several places, installedJavaVersion could be null or undefined.
parseInt could return NaN